src/components/Header/Header.js

```diff
@@ -40,23 +101,21 @@ export default function Header({
                 </HStack>
                 <HStack>
                   <FormControl>
-                    <Input
-                      placeholder="Workspace Name"
-                      value={workspace ?? ""}
-                      onChange={(e) => {
-                        saveWorkspace(e.target.value);
-                      }}
+                    <Select
+                      options={ allOrganizations.map( ( organization ) => ( { label: organization.name, value: organization.id } ) ) }
```

Comment:
You could consider creating a function called e.g. `entityToOption` and using it here and in the organizations select as well:

```js
function entityToOption({ name, id }) {
  return { label: name, value: id };
}
```

```suggestion
                      options={ allOrganizations.map(entityToOption) }
```

Reply:
Updated, thank you!